### ISSUE TITLE
Unreal Engine 4 fullscreen width squashed with Proton 7.0-1

### ISSUE THREAD

**Issue by minecraft2048** [ValveSoftware/Proton#5655]
# Compatibility Report
- Name of the game with compatibility issues: 
    -  Dragon Quest XI S (1295510)
    -  Octopath Traveler (921570)

## System Information
- GPU: RX 6700 XT
- Driver/LLVM version: Mesa 21.3.7-1
- Kernel version: 5.16.11-zen1-2-zen (possibly also Steam Deck kernel)
- Link to full system information report as [Gist](https://gist.github.com/): still can't copy full report because of https://github.com/ValveSoftware/steam-for-linux/issues/8124
- Proton version: 7.0-1
- Arch Linux 

## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [ ] that I have checked whether there are updates for my system available.

## Symptoms 
With Proton 6.3-8:
![20220307214536_1](https://user-images.githubusercontent.com/2445636/157018750-d2996c3d-5c5f-4073-9526-657674b1c477.jpg)

With Proton 7.0-1:
![20220307213929_1](https://user-images.githubusercontent.com/2445636/157018816-cf5cc6c9-b15b-43b7-87ff-a47b804bad07.jpg)

Same problem with DQXI S. My theory is that this regression is affecting UE4 games, to test that I'm downloading Ace Combat 7 
EDIT: Ace Combat 7 works

## Reproduction

1. Install these games with Proton 7.0-1 and above (including Experimental)
2. Revert Proton version to Proton 6.3-8 and see that those games work perfectly

## Proton logs

[steam-1295510.log](https://github.com/ValveSoftware/Proton/files/8200703/steam-1295510.log)
[steam-921570.log](https://github.com/ValveSoftware/Proton/files/8200722/steam-921570.log)

**Comment by kisak-valve** [comment]
Hello @minecraft2048, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.)

Related: https://github.com/ValveSoftware/Proton/issues/2511#issuecomment-1057967770

**Comment by alasky17** [comment]
This was a wayland specific regression and should be fixed with experimental-7.0-20220309.  Ty for reporting!

**Comment by DarkSwan86** [comment]
Hi! I'd like to add more information to this ticket, it seems to affect many other games that are not UE4 engine:

-Dead Space
-Mortal Kombat 9
-Bayonetta
-DOOM (2016)

My laptop fullscreen resolution is 1366x768, if i try it on all these games i have the same problem (the fullscreen window stretch very fast) but if i use a fullscreen resolution slightly smaller (1280x720) i have no such problem. Using Proton 6.3.8 also doesn't have this problem.

My system is a Kaby Lake laptop (Intel HD620), Mesa 21.3.7, kernel 5.15.10, Manjaro KDE up to date.

@alasky17 that version is already deployed on Steam using Proton Experimental?

**Comment by alasky17** [comment]
@DarkSwan86 Yes, current Proton Experimental should solve the problem - no need to use bleeding-edge :)  And please let me know if any of these games still exhibit the failure so that we can investigate further :) 

Edit: after re-reading your comment, it sounds like you might be running into a different bug, so I won't be too surprised if experimental doesn't fix this :/

**Comment by DarkSwan86** [comment]
@alasky17 ok i'll open a new ticket with the logs and other informations!
